mvpa2/generators/permutation.py

```diff
@@ -39,7 +39,7 @@ class AttributePermutator(Node):
     dataset.
     """
     def __init__(self, attr, count=1, limit=None, assure=False,
-                 strategy='simple', rng=np.random, **kwargs):
+                 strategy='simple', chunk_attr=None, rng=np.random, **kwargs):
```

Comment:
originally we used limit  to define grouping  here,  could/ should  it be used here as well  or you see a  use case  where  both would need to be specified